Educator Resources/Azure Workshop for Faculty/README.md

```diff
@@ -0,0 +1,105 @@
+# Microsoft Azure Skills Workshop for Faculty
+Welcome to Microsoft Azure Workshop Content Repo, explore Azure (cloud computing services) and it's Artificial Intelligence capabilities.
```

Comment:
Wording - needs grammar check.
Also, what exactly are you referring to by "Azure Workshop Content Repo"? That is not the name (or purpose) of this repo, so we shouldn't call it that.